libbeat/dashboards/import_dashboards.go

```diff
@@ -133,11 +147,27 @@ func New() (*Importer, error) {
 	}
 	indexSel := outil.MakeSelector(outil.FmtSelectorExpr(fmtstr, ""))
 
+	var tls *transport.TLSConfig
+
+	if len(cl.opt.Certificate) > 0 && len(cl.opt.Key) > 0 {
+		tls, err = outputs.LoadTLSConfig(&outputs.TLSConfig{
+			Certificate: outputs.CertificateConfig{
+				Certificate: cl.opt.Certificate,
+				Key:         cl.opt.Key,
```

Comment:
@andrewkroh Yes true, but as it's a temporary solution, I think it is better to minimize the changes.